src/layouts/BaseLayout/BaseLayout.test.tsx

```diff
@@ -212,49 +222,51 @@ describe('BaseLayout', () => {
     mockedUseImpersonate.mockReturnValue({ isImpersonating })
 
     server.use(
-      rest.get('/internal/user', (req, res, ctx) => {
-        return res(ctx.status(200), ctx.json(internalUser))
+      http.get('/internal/user', (info) => {
+        return HttpResponse.json(internalUser, { status: 200 })
+      }),
+      graphql.query('CurrentUser', (info) => {
+        return HttpResponse.json({ data: currentUser }, { status: 200 })
+      }),
+      graphql.query('DetailOwner', (info) => {
+        return HttpResponse.json({ data: mockOwner }, { status: 200 })
+      }),
+      http.get('/internal/:provider/:owner/account-details', (info) => {
+        return HttpResponse.json({}, { status: 200 })
       }),
-      graphql.query('CurrentUser', (_, res, ctx) =>
-        res(ctx.status(200), ctx.data(currentUser))
-      ),
-      graphql.query('DetailOwner', (_, res, ctx) =>
-        res(ctx.status(200), ctx.data(mockOwner))
-      ),
-      rest.get('/internal/:provider/:owner/account-details', (_, res, ctx) =>
-        res(ctx.status(200), ctx.json({}))
-      ),
       // Self hosted only
-      graphql.query('HasAdmins', (_, res, ctx) =>
-        res(ctx.status(200), ctx.data({}))
-      ),
-      graphql.query('Seats', (_, res, ctx) =>
-        res(ctx.status(200), ctx.data({}))
-      ),
-      graphql.query('TermsOfService', (_, res, ctx) =>
-        res(ctx.status(200), ctx.data({}))
-      ),
-      graphql.query('UseMyOrganizations', (_, res, ctx) =>
-        res(
-          ctx.status(200),
-          ctx.data({
-            myOrganizationsData: {
-              me: {
-                myOrganizations: {
-                  edges: [],
-                  pageInfo: { hasNextPage: false, endCursor: 'MTI=' },
+      graphql.query('HasAdmins', (info) => {
+        return HttpResponse.json({ data: {} }, { status: 200 })
+      }),
+      graphql.query('Seats', (info) => {
+        return HttpResponse.json({ data: {} }, { status: 200 })
+      }),
+      graphql.query('TermsOfService', (info) => {
+        return HttpResponse.json({ data: {} }, { status: 200 })
+      }),
+      graphql.query('UseMyOrganizations', (info) => {
+        return HttpResponse.json(
+          {
+            data: {
+              myOrganizationsData: {
+                me: {
+                  myOrganizations: {
+                    edges: [],
+                    pageInfo: { hasNextPage: false, endCursor: 'MTI=' },
+                  },
                 },
               },
             },
-          })
+          },
+          { status: 200 }
         )
-      ),
-      graphql.mutation('updateDefaultOrganization', (req, res, ctx) =>
-        res(ctx.status(200), ctx.data({}))
-      ),
-      rest.get('/internal/users/current', (_, res, ctx) =>
-        res(ctx.status(200), ctx.json({}))
-      )
+      }),
+      graphql.mutation('updateDefaultOrganization', (info) => {
+        return HttpResponse.json({ data: {} }, { status: 200 })
```

Comment:
In App.test.tsx you added `{ status: 200 }` as an example of setting status which I'm cool with, but since 200 is default, IMO we shouldn't make explicitly stating it a pattern across the codebase. Thoughts?

Reply:
Yea we can remove these

Reply:
tyty appreciate